### DISCUSSION TITLE
I'm facing difficulties and wondering if there's any documentation on using DataGrid without MVVM.

### DISCUSSION THREAD

**Discussion by IronEaterBeast** [AvaloniaUI/Avalonia#14091] [Q&A] [answered]

**Comment by thevortexcloud** [comment]
There is an example in the samples

https://github.com/AvaloniaUI/Avalonia/blob/2385395ed566948612a92e8b36d1c841f6ec6ba5/samples/ControlCatalog/Pages/DataGridPage.xaml.cs

**Comment by timunie** [comment]
You can use an ObservableCollection or List (will not auto-update) anywhere in your code. could even be a static property. However I highly suggest to consider the MVVM approach even for a smaller App, as it has a lot of benefits. Avalonia.Samples may help to onboard you, if that concept is new to you.

**Reply by timunie** [reply, marked as answer]
make sure you have the package installed and added the needed Styles. https://docs.avaloniaui.net/docs/reference/controls/datagrid/

**Reply by IronEaterBeast** [reply]
Dam, you are right.
